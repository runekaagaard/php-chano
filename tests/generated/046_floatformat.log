
---- EXPECTED OUTPUT
6.200
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/046_floatformat.php on line 4
---- FAILED
