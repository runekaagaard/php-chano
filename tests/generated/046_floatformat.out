Fatal error: Call to undefined method DtlIter::floatformat() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/046_floatformat.php on line 4